packages/ag-charts-community/src/chart/data/dataModel.ts

```diff
@@ -828,6 +885,28 @@ export class DataModel<
             }
         };
 
+        const { columns } = processedData;
+        if (columns != null && processedData.type === 'grouped') {
+            for (const group of processedData.groups) {
+                for (const processor of groupProcessors) {
+                    if (group.validScopes) continue;
+
+                    const valueIndexes = groupProcessorIndices.get(processor) ?? [];
+                    const adjustFn = groupProcessorInitFns.get(processor)?.();
+
+                    if (!adjustFn) continue;
+
+                    for (const index of group.indices) {
+                        adjustFn(undefined!, valueIndexes, index, columns);
+                    }
+                }
+
+                // for (const index of group.indices) {
+                //     updateDomains(values);
+                // }
```

Comment:
Nit: Needed?

Reply:
Eventually. The bar series doesn't yet work if you turn off the row model